Makefile

```diff
@@ -5,6 +5,9 @@ GO = go
 BUILD_FLAGS = -gcflags '-N -l'
 BUILD_QUIET ?= -q
 
+GOLANG_IMAGE ?= golang:1.14
```

Comment:
golang image is configured in multiple places - like in the dockerfiles used in `make images`
Do you plan to change all the places?

Reply:
Do u mean using build args?

Reply:
Sorry I take it back. This is good then.